doc/rst/source/gmt.conf.rst

```diff
@@ -1151,10 +1151,13 @@ fonts can be found in the :doc:`gmt` man page.
 .. _PS_MEDIA:
 
 **PS_MEDIA**
-    Sets the physical format of the current plot paper [a4 (or letter)].
-    The setting only applies to PostScript plots (i.e., classic mode or when using
-    modern mode and selecting PostScript outout).  The following formats (and their
-    widths and heights in points) are recognized:
+    Sets the physical size of the current plotting paper. This is required
```

Comment:
This paragraph is a little hard to understand. Could you rephrase it?

Reply:
OK, done